simulation-construction-set-tools/src/test/java/us/ihmc/simulationConstructionSetTools/util/simulationrunner/GoalOrientedTestConductor.java

```diff
@@ -257,7 +254,7 @@ public void concludeTesting(String videoName)
 
       if (simulationTestingParameters.getCreateSCSVideos())
       {
-         BambooTools.createVideoWithDateTimeAndStoreInDefaultDirectory(scs, videoName);
+         // TODO GITHUB WORKFLOWS
```

Comment:
see previous comment on making it so this doesn't disappear by just adding a deprecated referecne that can be found more easily .